docs/gen-docs/kyma_run_function.md

```diff
@@ -0,0 +1,40 @@
+---
+title: kyma run function
+---
+
+Run functions locally.
+
+## Synopsis
+
+Use this command to run function in docker from local sources.
+
+```bash
+kyma run function [flags]
+```
+
+## Options
+
+```bash
+      --containerName string   The name of the created container.
+  -d, --detach                 Change this flag to true if you don't want to follow the container logs after run'.
+  -e, --env stringArray        The system environments witch which the container will be run.
+  -f, --filename string        Full path to the config file.
+      --imageName string       Full name with tag of the container.
+  -p, --port string            The port on which the container will be exposed. (default "8080")
```

Comment:
```suggestion
  -p, --port string            The port on which the container will be exposed. (default "8080").
```

Reply:
`(default "8080")` documentation generator adds this dot :D

Reply:
Sorry, my bad.